### PR TITLE
API-14291  provider dashboard  api endpoint status (loast summary)

### PR DESCRIPTION
**Pull request by mheusgit**
- Added new LOAST controller with minimal functionality to get the ball rolling
- Change includes "summary.ts".  File is filled with sample LOAST testing data to aid with early development

### CODE REVIEW

File: src/loast/loast.controller.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      this.listResults,
+    );
+
+    this.apiVersionController.registerSubRoutes(this.path, this.subRouter);
+  }
+
+  listResults: RequestHandler = (request, response, next) => {
+    try {
+      const resultList = summaryData;
+      response.json(resultList);
```
Comment (dsteeve) [inline src/loast/loast.controller.ts:50]: maybe add a clear "TODO" to call the loastService to get the data from the db once that is in place. For now just getting mocked data for UI development

**Review by dsteeve — APPROVED** [review]
My comments are pretty minor.  Just perhaps a little more clarity around the mock data and question on auth needs.

File: src/loast/loast.controller.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import apiVersionResolverMiddleware from '../middleware/apiVersionResolver.middleware';
+import APIVersionService from '../api/version/apiVersion.service';
+import APIVersionController from '../api/version/apiVersion.controller';
+import { apiVersionLookupSchema } from '../api/version/apiVersion.schema';
+import summaryData from './loastResult/summary';
+
+class LoastController {
+  public path = '/loast';
+  public router = Router();
+  public authRouter = Router();
```
[the remark this answers is not in the document]
Comment (mheusgit) [reply src/loast/loast.controller.ts:14]: Will be needed in the near future to support POST: /docserver/apis/<NAME>/versions/<VERSION>/loast/results

File: src/loast/loast.controller.ts (hunk 1 above)
Comment (mheusgit) [reply src/loast/loast.controller.ts:52]: Its added

**Review by ediiotero — APPROVED** [review]
looks good to me

**Pull request merged**